source/chapters/effects.rst

```diff
@@ -27,6 +27,9 @@ processed one after the other in series. You will hear different sounds
 depending on the order of effects within the chain. Each effect in the chain
 has its own button to toggle it on and off.
 
+The shortcut to quickly Activate/Disable Effects Unit is 5 Deck1 and 0 in Deck2 
+in an English keyboard layout.
```

Comment:
Which effect unit is activated by each of these shortcuts?

Also, I think this would be better to put at the bottom of the section after the signal flow through the effect units is explained.

Reply:
Those are the QuickEffect shortcuts, so wrong context.
addressed by later review.